src/main/java/com/minecolonies/proxy/ClientProxy.java

```diff
@@ -20,6 +23,7 @@
 import net.minecraft.client.resources.model.ModelResourceLocation;
 import net.minecraft.item.Item;
 import net.minecraft.util.BlockPos;
+import net.minecraftforge.client.ForgeHooksClient;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraftforge.client.ForgeHooksClient'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)